packages/SwingSet/tools/paths.js

```diff
@@ -0,0 +1,11 @@
+import url from 'node:url';
+import { resolve } from 'import-meta-resolve';
+
+/**
+ * @param {string} specifier
+ * @returns {string} file system absolute path of the package resource specifier
+ */
+export const pkgAbsPath = specifier => {
```

Comment:
It's not clear that this is only used with paths that are known at design-time. Let's keep runtime I/O async.